build/changelog/entries/2019/07/12519.SUP-6751.enhancement

```diff
@@ -0,0 +1,37 @@
+The generic content handler now allows configuration to transform elements with specific attributes into other elements.
+This is used to counter Chrome 75's copy paste issues in contenteditable.
+
+Chrome will try to maintain styling on the copied range.
+Depending on the styling of the copied element this may lead to `<b>` elements turning into `<span style="font-size: 700">`
```

Comment:
It's been quite a while, but I thought the CHANGELOG is [Textile](https://devhints.io/textile) and not Markdown. Did you try to build the CHANGELOG?

Reply:
thank you for pointing this out changed backticks to `@` and `<javascript>` respectively